src/services/mongodb/collection.js

```diff
@@ -143,109 +143,56 @@ Collection.prototype.upsert =
 
 // private
 function insertOp(self, docs, options) {
```

Comment:
we already discussed offline, but summary of changes to make here for the new crud stuff:

- `insert` is now split into `insertOne` and `insertMany`. 
    - the payload for `insert` should look like `{database: "db", collection: "collection", document: { ... doctoinsert}}`
    - the payload for `insertMany` should look like `{database: "db", collection: "collection", documents: [doc1, doc2, doc3]}`. However note that `insertMany` is *not* variadic, its argument should be an array.

- the results returned by each CRUD action are as described here: https://jira.mongodb.org/browse/STITCH-902. This should be useful in the integration tests.